### ISSUE TITLE
Console.log message does not display 

### ISSUE THREAD

**Issue by sandertyu** [LibreTexts/ckeditor-binder-plugin#139]
On release [1.6.0](https://github.com/LibreTexts/ckeditor-binder-plugin/tree/1.6.0) we are using a [github workflow](https://github.com/LibreTexts/ckeditor-binder-plugin/blob/master/.github/workflows/main.yml) to insert information about the version number and date into a string in [index.js](https://github.com/LibreTexts/ckeditor-binder-plugin/blob/master/src/scripts/index.js#L12) which is then printed out to console with console.log. This action is only activated when merging to master, but our most recent merge does not show the console message at all. I've created test branch [sed-test](https://github.com/LibreTexts/ckeditor-binder-plugin/tree/sed-test) which has a folder `sed-test` with a script `sed.sh` to recplicate the code in the github workflow `main.yaml` but only on `.txt` files. There, the insertion process works properly. Additionally, I have verified that printing to console works from within `index.js` but you can test for yourself (sed-test prints through `activateThebelab`). I use both `bin/deploy.sh sed-test` to test the branch and `yarn build && yarn start` to test locally (sed-test is currently on query.libretexts.org).

The issue is most likely happening during the process of the github action, but we'll need a way to test that without using the master branch. Even if sed were not properly inserting, the console message should still print a string, so the whole message is being messed up.

**Comment by moorepants** [comment]
Have you looked at the index.js code which is deployed? Are the lines changed and edited like you are hoping?

**Comment by sandertyu** [comment]
index,js was not displaying in the webpack while we were printing through index.js, so I wasn't able to check. I also think I made the mistake of not actually deploying the proper branches to query when testing the console messages, it might have been due to some sort of cacheing. Hopefully printing the message now will make mistakes like this harder to glance over. 

anyways, this should be solved by https://github.com/LibreTexts/ckeditor-binder-plugin/pull/140
